javascript/index.js

```diff
@@ -14,52 +14,85 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+    printMinutes();
+    printSeconds();
+    // printMilliseconds();
 }
 
 function printMinutes() {
-  // ... your code goes here
+    const mins = chronometer.twoDigitsNumber(chronometer.getMinutes());
+    minDec.innerText = mins[0];
+    minUni.innerHTML = mins[1];
 }
 
 function printSeconds() {
-  // ... your code goes here
+    const secs = chronometer.twoDigitsNumber(chronometer.getSeconds());
+    secDec.innerText = secs[0];
+    secUni.innerHTML = secs[1];
 }
 
 // ==> BONUS
-function printMilliseconds() {
-  // ... your code goes here
-}
+// function printMilliseconds() {
+//     const millis = chronometer.twoDigitsNumber(chronometer.getMilliseconds());
+//     milDec.innerText = millis[0];
+//     milUni.innerText = millis[1];
+// }
 
 function printSplit() {
-  // ... your code goes here
+    let list = document.createElement('li');
+    list.innerHTML = chronometer.setTime();
+    parent.appendChild(list);
 }
 
 function clearSplits() {
-  // ... your code goes here
+    document.getElementById("splits").innerHTML = "";
 }
 
 function setStopBtn() {
```

Comment:
The logic defined line 52-54 is for setting the "start" button, therefor you should either rename your function or switch the logic with function setStartBtn